docs/UserGuide.md

```diff
@@ -102,7 +105,7 @@ Format: `findo KEYWORD [MORE_KEYWORDS]`
 * Only the name is searched.
 * Only full words will be matched e.g. `keychains` will not match `keychain`
 * Orders matching at least one keyword will be returned (i.e. `OR` search).
-  e.g. `apple keychain` will return `apple painting`, `banana keychain`
+  e.g. `findo apple keychain` will return `apple painting`, `banana keychain`
```

Comment:
for this, since i think most of the ug puts examples after a line break, perhaps you can add it also here?